database/src/test/scala/za/co/absa/atum/database/runs/GetPartitioningByIdIntegrationTests.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.database.runs
+
+import za.co.absa.balta.DBTestSuite
+import za.co.absa.balta.classes.JsonBString
+import io.circe.Json
+import io.circe.parser._
+
+class GetPartitioningByIdIntegrationTests extends DBTestSuite {
+
+  private val fncGetPartitioningById = "runs.get_partitioning_by_id"
+
+  private val partitioning = JsonBString(
+    """
+      |{
+      |  "version": 1,
+      |  "keys": ["key1", "key3", "key2", "key4"],
+      |  "keysToValues": {
+      |    "key1": "valueX",
+      |    "key2": "valueY",
+      |    "key3": "valueZ",
+      |    "key4": "valueA"
+      |  }
+      |}
+      |""".stripMargin
+  )
+
+  private val expectedPartitioning: Json = parse(partitioning.value) match {
```

Comment:
maybe it could be even simplified since we don't transform the success value in any way
```scala
parse(partitioning.value).getOrElse(throw new Exception("Failed to parse JSON"))
```